modules/candidate_parameters/README.md

```diff
@@ -0,0 +1,50 @@
+# Imaging Browser
+
+## Purpose
+
+The Candidate Parameters module allows users to input 
+candidate data that is not timepoint specific.
+
+## Intended Users
+
+The primary types of users are:
+1. Data-entry personnel
+2. Site coordinators and data-monitors validating the data
```

Comment:
I'm not sure what a data-monitor is.

Maybe we should have a standardized list of roles/types of users for these docs?